lib/cadet/stories/assets.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Cadet.Stories.Assets do
```

Comment:
This file would probably be better named `lib/cadet/assets/assets.ex` or something&mdash;it's not inherently specific to the game or story.